libse/SubtitleFormats/UnknownSubtitle15.cs

```diff
@@ -38,8 +38,7 @@ private static string ToTimeCode(TimeCode tc)
         private static TimeCode DecodeTimeCode(string s)
         {
             var parts = s.Split(new[] { ';', ':' }, StringSplitOptions.RemoveEmptyEntries);
-            var tc = new TimeCode(int.Parse(parts[0]), int.Parse(parts[1]), int.Parse(parts[2]), int.Parse(parts[3]) * 100);
-            return tc;
```

Comment:
> *100 is now missing!?

Nope, check the line bellow. In here I just returned instead of set var and return